src/resources/js/pages/sessions/register/info.vue

```diff
@@ -240,6 +240,7 @@
                                                                         testCase.name
                                                                     }}
                                                                 </span>
+                                                                <span class="btn btn-outline-primary" v-if="!testCase.public">Not Public</span>
```

Comment:
```suggestion
                                                                <span class="badge badge-primary" v-if="!testCase.public">Not Public</span>
```

`.btn` looks too clickable, which is misleading because nothing happens if you click. https://getbootstrap.com/docs/4.0/components/badge/